.stickler.yml

```diff
@@ -0,0 +1,7 @@
+linters:
+  eslint:
+    config: '.eslintrc.js'
+
+files:
+  ignore:
+    - 'node_modules'
```

Comment:
Why are node modules ignored? They shouldn't be in GitHub to begin with.

Reply:
Oh that was from an example. You're right, that ignore is redundant. I'll take it out